.nfpm_template.yaml

```diff
@@ -0,0 +1,20 @@
+# nfpm configuration file used as template.
```

Comment:
Files names use a different format of what we have for tar files:

`ig_<version>_<arch>.deb|rmp`, e.g.,  ig_0.14.0_amd64.deb

vs

`ig-<os>-<arch>-v<version>.tar.gz`, e.g., ig-linux-amd64-v0.14.0.tar.gz

IMO, they must follow the same naming format.